src/main/java/org/prebid/server/bidder/appnexus/AppnexusBidder.java

```diff
@@ -312,7 +312,7 @@ private ImpWithMemberId makeImpWithMemberId(Imp imp, String defaultDisplayManage
         }
 
         final BigDecimal reserve = appnexusExt.getReserve();
-        if (reserve != null && reserve.compareTo(BigDecimal.ZERO) > 0) {
+        if (bidFloorIsInvalid(imp.getBidfloor()) && reserve != null && reserve.compareTo(BigDecimal.ZERO) > 0) {
```

Comment:
Can we create something like `isValidBidfloor` and then use it both for imp.getBidfloor() and reserve?